### PR TITLE
FIT-5 Test bubble perryv2

### PR DESCRIPTION
**Pull request by ratneshraval**
#### Which User story does this relate to?
- http://jen-proxy.dev.cwds.io/job/cals-app-pull-request/81/ 

#### Brief feature description
- Test bubble for cals with Perry v2

### CODE REVIEW

**Comment by anuroop15** [comment]
jenkins re-test this please

**Comment by ratneshraval** [comment]
retest this please

File: docker/cals-test-bubble/docker-compose.env [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
+API_PORT=8081
 LOGIN_URL=http://localhost:${PERRY_PORT}/perry/authn/login
 LOGOUT_URL=http://localhost:${PERRY_PORT}/perry/authn/logout
 SHIRO_CONFIG_PATH=/opt/cals-api/config/shiro.ini
 KEY_STORE_FILE=/opt/cals-api/config/testKeyStore.jks
-
+SWAGGER_JSON_URL=http://localhost:${API_PORT}/swagger.json
+SHOW_SWAGGER=true
+SWAGGER_CALLBACK_URL=http://localhost:${API_PORT}/swagger
+SWAGGER_TOKEN_URL=http://perry:8080/perry/authn/token
```
Comment (anuroop15) [inline docker/cals-test-bubble/docker-compose.env]: is there a change in swagger_token_url port too?? or is it still 8080?

Comment (ratneshraval) [reply docker/cals-test-bubble/docker-compose.env]: That is Perry port so 8080 is correct.

**Pull request merged**